src/jdmtool/featunlk.py

```diff
@@ -448,21 +497,28 @@ def display_content_of_dat_file(dat_file: pathlib.Path):
 
         name = header_bytes[0x49:0x49+20].decode('ascii')
         print(f'** {name}')
+        cycle = header_bytes[0x59:0x59+4].decode('ascii')
+        print(f'** Cycle: {cycle}')
         description = header_bytes[0x65:0x83].decode('ascii')
         if description.strip():
             print(f'** {description}')
         year = int.from_bytes(header_bytes[0x39:0x39+2], 'little')
         month = int(header_bytes[0x3B])
         day = int(header_bytes[0x3c])
-        print(f'** Creation Date: {year}-{month:02}-{day:02}')
+        cus_date1 = (datetime.strptime(f'{day:02}{month:02}{year}', "%d%m%Y").date()).strftime("%d-%b-%Y")
+        print(f'** Creation Date: {cus_date1}')
 
         release = int.from_bytes(header_bytes[0x87:0x89], 'little')
-        print(f'** Release: {release}')
 
         if int.from_bytes(header_bytes[0x83:0x85], 'little') == 0xDEAD:
             version = str(header_bytes[0x85]) + '.' + str(header_bytes[0x86])
             release = int.from_bytes(header_bytes[0x87:0x89], 'little')
             print(f'** Creation Software Version: {version} ({release})')
+    elif feature in (Feature.SECTIONALS,):
+        print('** Cycle: ' + header_bytes[101:101+4].decode('ascii'))
+        cus_date1 = (datetime.strptime(f'{header_bytes[171:171+10].decode('ascii')}', "%m/%d/%Y").date()).strftime("%d-%b-%Y")
```

Comment:
`strptime` is fine here since you actually need to parse a string - just change the `strftime`